config/crd/bases/secret-agent.secrets.forgerock.io_secretagentconfigurations.yaml

```diff
@@ -69,6 +71,7 @@ spec:
                   - AWS
                   type: string
               required:
+              - azureVaultName
```

Comment:
This doesn't seem like it should be required. E.G. if you're using GCP Secret Manager, you shouldn't need to set Azure values. Instead, we can use the custom validation logic to make sure it's set if you selected Azure and the SM.

BTW, this file is not supposed to be modified directly, but instead via the _types.go file.